addons/semicolon/addon.json

```diff
@@ -20,5 +32,13 @@
     }
   ],
   "tags": ["easterEgg"],
-  "enabledByDefault": false
+  "enabledByDefault": false,
+  "settings": [
+    {
+      "name": "Invert the colors of the semicolon (black background and white semicolon). Useful with the Website dark mode addon",
+      "id": "inverted",
+      "type": "boolean",
+      "default": false
+    }
+  ]
```

Comment:
well, the original bug wasn't with the inverted colors. ofc the original site also doesn't have dark mode. so I think the user should be able to choose.

Reply:
I don't think you get it... all changes should happen in the dark mode CSS. It is responsible in the first place... so there should be no setting because it is intended.